### ISSUE TITLE
Usernames not being read by screen readers like they used to be

### ISSUE THREAD

**Issue by jfbosch** [bitwarden/clients#12431]
### Steps To Reproduce

1. Turn on a screen reader, such as Windows Narrator or Zoomtext.
2. Go to a site for which you have an entry in the BitWarden vault. 
3. Open the browser extension.
4. Press Tab to navigate down until you reach the vault entry relating to the current site.

### Expected Result

I expected the screen reader to read both the name of the entry, followed by the username.
In the below example, it should have read, "github.com jfbosch".

![image](https://github.com/user-attachments/assets/5b0bab5f-2fe7-4211-9ba4-fcf45da9b0d8)

### Actual Result

The screen reader only reads the name of the entry, and not the username as well, even though the username is shown on the item under the entry name.
In my example, it only reads "github.com" and not also "jfbosch".

### Screenshots or Videos

![image](https://github.com/user-attachments/assets/262e0d9e-baf2-4ddf-b05d-dbc9377a1cea)

### Additional Context

The result is, if I have more than one login for the same site, in other words, with different usernames, I cannot tell which is which by using the screen reader, making it inaccessible to blind people. 
This used to work fine in the previous version of the extension. 

### Operating System

Windows

### Operating System Version

11

### Web Browser

Microsoft Edge

### Browser Version

Version 131.0.2903.99 (Official build) (64-bit)

### Build Version

2024.12.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by cbbit** [comment]
Hi there,

This report has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by jfbosch** [comment]
Hi. Glad to hear it has been escalated. 
I believe I have provided all the relevant info, But feel free to reach out if there is anything I can assist with.

**Comment by vincentsalucci** [comment]
This has been fixed with #13800 and will be released later this month! Thanks for your help!
